plop/collector.py

```diff
@@ -182,18 +183,19 @@ def main():
                 # interpreted as relative to this module.
                 global __package__
                 del __package__
-                exec f.read() in globals(), globals()
-    except SystemExit, e:
+                exec(f.read() in globals(), globals())
```

Comment:
This doesn't work (it will try to `exec` a bool returned by the `in` expression).